src/RichText/RichText.php

```diff
@@ -346,16 +346,26 @@ private static function fixImagesPath(string $content): string
             "/ (href)='[^']*\/front\/document\.send\.php([^']+)' /",
 
             // src attribute, surrounding by " or '
-            '/ (src)="[^"]*\/front\/document\.send\.php([^"]+)" /',
-            "/ (src)='[^']*\/front\/document\.send\.php([^']+)' /",
+            '/ (src)="(?!data:)[^"]*\/front\/document\.send\.php([^"]+)" /',
+            "/ (src)='(?!data:)[^']*\/front\/document\.send\.php([^']+)' /",
```

Comment:
> This does not seems related to a deprecated method

The fix completely changed since the creation of the PR. The title/description should be updated.

Anyway, having `data:` URLs here is unexpected and indicates that something goes wrong previously during the `CommonDBTM::addFiles()` execution, or maybe the `CommonDBTM::addFiles()` method was not executed, or maybe the issue is related to a know issue in GLPI when big images are uploaded (see #13606) and that has been fixed in GLPI 11.0.

Bypassing the issue by filtering the images with a `data:` source is OK, but I think there is an issue somewhere else to identify.